app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -169,6 +169,12 @@ object MapScreen {
    */
   const val USER_LOCATION_MARKER_ICON_SIZE = 40
 
+  /**
+   * (Config) Size of the icon representing the starting point of a hike on the map. The size is
+   * defined empirically to make the icon visible and not too big.
+   */
+  const val HIKE_STARTING_MARKER_ICON_SIZE = 40
```

Comment:
Question: What unit is this? Is it in pixels? If so, how does it render on small screens?

Reply:
I did this the same way as the user location marker. It seems that both render the same in all conditions.
I've just checked on a small phone. We may see it too big but I think it's fine like this. See the image:

<img width="375" alt="image" src="https://github.com/user-attachments/assets/ae02a141-47a3-4fec-9afd-e57e0af70436" />

Reply:
Mmmmh, yeah, I agree